markdown/events/2022/bytesize_nf-core-airrflow.md

```diff
@@ -24,3 +24,87 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[12:29](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=749)
+After alignment to the IMGT reference there is a number of quality filtering steps that are performed. First it's checked that the locus matches exactly the V_calls. The V(D)J segment assignment, that there are a minimum of 200 informative positions, maximum 10% and nucleotides, that the sequences that are determined are productive, that the junction region is a multiple of three amino acids. There's also a possibility of removing chimeric reads, detecting contamination across samples and finally collapsing duplicate sequences if there are any. Plenty of quality filters as part of the pipeline.
+
+[13:18](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=978)
+The next step is clonal analysis. In that case hierarchical clustering is used based on the hamming distances between sequences. The pipeline is also able to auto-detect a hamming distance threshold that can be used to determine, which sequences are part of the same clone, or are part of different clones. There is also a step for lineage reconstruction of the clonal lineage trees. Recently the pipeline offers to use the Enchanter tool, which is developed by Susanna Marcus in immcantation. That tool provides calls to other immcantation tools and also nice reports for each of these steps. I invite you to check them out here.
+
+[14:05](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=845)
+Finally, there's a repertoire analysis step. An R Markdown report is provided, that summarises the repertoire analysis results for all samples. Of notice is that a custom R Markdown report can also be provided, in case that the user wants to change some things in this report. It's also possible to provide an R Markdown file. Other reports of this reporting analysis steps is the MultiQC QC report for all samples from the grid quality control reports.
+
+[14:43](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=883)
+Here we will see an example of this repertoire analysis report. First there is a summary of all the samples used for the analysis. Clonal abundance and clonal diversity are reported, together with vision usage. Finally all of the tools that are used as part of the pipeline and their citations are noted here, to make it really easy for users of the pipeline to also cite the original tools, that are being used.
+
+[15:12](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=912)
+As you know, all documentation for nf-core/airrflow can be found on the nf-core website, so check it out if you want to use it. There are also some example results of the pipeline when the full tests are run on AWS.
+
+[15:30](https://www.youtube.com/watch?v=CrJgxVRVlqY&t=930)
+What's next for this pipeline? Stay tuned for a new release that comes really soon - we hope this week or the next. It will includes more quality control and reporting as part of the Enchanter tool, as I have mentioned, by Susanna Marquez, as well as and code refactoring using subworkflows.
```

Comment:
```suggestion
What's next for this pipeline? Stay tuned for a new release that comes really soon - we hope this week or the next. It will includes more quality control and reporting as part of the EnchantR tool, as I have mentioned, by Susanna Marquez, as well as and code refactoring using subworkflows.
```